core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/Entitlements.java

```diff
@@ -155,6 +161,20 @@ public static EntitlementClassesEnum of(EntitlementClass<?> entitlementClass) {
         }
     }
     
+    public static class TaskAndItem<T> extends Pair<Task<?>,T> {
+        public static <TT> TypeToken<TaskAndItem<TT>> typeToken(Class<TT> type) {
+            return new TypeToken<Entitlements.TaskAndItem<TT>>() {
```

Comment:
Is the `TT` type actually included in the anonymous class? I'd expect not since the information is not available at compile time, so no use of the `type` argument. Just curious, I see it's the same in similar pairs.